sdk/purview/azure-purview-workflow/azure/purview/workflow/_client.py

```diff
@@ -102,7 +103,7 @@ def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any)
         self.approval = ApprovalOperations(self._client, self._config, self._serialize, self._deserialize)
         self.task_status = TaskStatusOperations(self._client, self._config, self._serialize, self._deserialize)
 
-    def send_request(self, request: HttpRequest, **kwargs: Any) -> HttpResponse:
+    def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs: Any) -> HttpResponse:
```

Comment:
Did we regenerate to get these changes?

Reply:
Yes, the pylint/pyright errors were in the generated code so I used he swagger README.md to regenerate the library